node/runtime/src/lib.rs

```diff
@@ -65,8 +66,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("node"),
 	impl_name: create_runtime_str!("substrate-node"),
 	authoring_version: 10,
-	spec_version: 26,
-	impl_version: 26,
+	spec_version: 27,
```

Comment:
You don't need to increase the `spec_version`. `impl_version` should be enough. CI will still report an issue, but that is okay for the runtime check.